EpicLoot/src/Magic/MagicItemEffects/ModifySummonDamage.cs

```diff
@@ -0,0 +1,123 @@
+using System.Collections.Generic;
+using HarmonyLib;
+using UnityEngine;
+
+namespace EpicLoot.MagicItemEffects;
+
+public class ModifySummonDamage
```

Comment:
Similar to ModifyWispRange, reduce if checking nesting here by returning.

Reply:
Fixed in: https://github.com/OrianaVenture/Randy_Vapok_ValheimMods/pull/10/commits/a21b532315f05a12626340d37fb68b91c5fbdf63